survey-runner/terraform.tfvars.example

```diff
@@ -1,27 +1,26 @@
-aws_access_key="XXXXXXXXXXXX"
-aws_secret_key="XXXXXXXXXXXX"
-aws_key_pair="XXXXX"
-elastic_beanstalk_aws_key_pair="XXXXX"
-ons_access_ips=[XXXXXXX, XXXXXXX]
-certificate_arn="arn:aws:iam::XXXXXXX:server-certificate/NAME"
+aws_access_key="XXXXXXXXXXXX" # Update with access key
+aws_secret_key="XXXXXXXXXXXX" # Update with secret key
+aws_key_pair="XXXXX" # Set to the name of pem file (Not including .pem)
+elastic_beanstalk_aws_key_pair="XXXXX" # Set to the name generated ssh key in aws
+ons_access_ips=[XXXXXXX, XXXXXXX] # Set to the IP addresses you will be accessing accessing survey runner from
+certificate_arn="arn:aws:iam::XXXXXXX:server-certificate/NAME" # Set to arn of the tls certificate used by survey runner
 application_secret_key="you'll never guess it"
 vpc_ip_block="10.30.20.0/24"
-rabbitmq_ip_prime="XX.XX.XX.15"
-rabbitmq_ip_failover="XX.XX.XX.16"
+rabbitmq_ip_prime="10.30.20.15"
+rabbitmq_ip_failover="10.30.20.16"
 google_analytics_code="UA-56892037-7"
```

Comment:
I think the default value for `google_analytics_code` should be `""`. That should mean that eq-survey-runner doesn't setup analytics (https://github.com/ONSdigital/eq-survey-runner/blob/master/app/__init__.py#L113)
Also it's duplicated below set to `XXXXXXX`